monitoring/update-bucket-capacity-info-cronjob/alerts.yaml

```diff
@@ -0,0 +1,99 @@
+# Variables which should be replaced. Similar to grafana dashboards' __inputs section
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: update_bucket_capacity_info_cronjob
+    type: constant
+    value: artesca-data-ops-update-bucket-capacity-info
+  - name: update_bucket_capacity_info_job_duration_threshold
+    type: config
+    value: 240
```

Comment:
4m is the interval of update-bucket-capacity-info cronjob, will alert if a job runs longer than this duration